### DISCUSSION TITLE
Like/Love song button and Like/love playlist

### DISCUSSION THREAD

**Discussion by naymapl** [jpochyla/psst#237] [Ideas]
Hi.
I build native arm64 apple silicon app and runing great. I need to build it like a app right now but app runing so great and use low memory compare to m1 spotify app. But Your app is missing one small important for me item: like/love button.

So when you listen weekly playlist I many times click like/love song button and this song go automaticly to like/love playlist in spotify - but this two thinks is missing in psst app. Any chance to add this in feature ?

Thanks a lot for answer.

**Comment by jpochyla** [comment]
Hi, you can right-click a track and select "Save Track to Library". Do you mean something else?

**Reply by naymapl** [reply]
@jpochyla hey. Thanks for reply. Your solution is ok but I mean something difrent.
In original Spotify app you have heart icon and when you click it its mean you love it.  
![Zrzut ekranu 2021-12-20 o 21 27 39](https://user-images.githubusercontent.com/24625283/146828657-7f4f9c3f-c488-4104-9daf-a07aca119d71.jpg)
But is going deeper. When you listen to weekly songs playlist and love track algoritm of Spotify know what king of music you like and give you more next week. All this song are in special playlist:
![Zrzut ekranu 2021-12-20 o 21 27 58](https://user-images.githubusercontent.com/24625283/146828877-16ea1389-7575-4145-98a9-61e4660c9571.jpg)

This one with heart - but you can`t access this playlist from Psst. Its generate automaticly by Spotify i guest.

Hope so you understand what I mean - I`m not perfect in english language.

**Reply by jpochyla** [reply]
The playlist is available under Tracks in the main menu.

**Reply by naymapl** [reply]
@jpochyla sorry but is not there. 

![Zrzut ekranu 2021-12-21 o 09 00 30](https://user-images.githubusercontent.com/24625283/146893142-bc441906-edd9-4169-9843-13e36bb3e9c9.jpg)

I have just **Discover Weekly** playlist but no **Loved Songs**.

**Reply by jpochyla** [reply]
**Tracks**, right under Home.

**Reply by naymapl** [reply]
@jpochyla ok - i got it :D Tracks is **Loved Tracks/Songs** and You need just click right on any track and add it to Tracks - is this same like click on ❤️ in orginal Spotify 👍 Thank you so much for explain how is work. Great. Have a nice day and Merry Christmas for you !!

**Reply by jpochyla** [reply]
Haha glad we solved it! Heart icon would be useful though, we might add it in the future. Happy holidays to you as well!

**Reply by naymapl** [reply]
That for sure will good for begining users and more user friendly.

I wrote small article about Psst: https://blog.nayma.pl/psst-lekki-i-natywny-klient-spotify-dla-apple-silicon/ Hope so you will like it.
